gradio/interface.py

```diff
@@ -96,7 +96,7 @@ def __init__(
         inputs: str | Component | list[str | Component] | None,
         outputs: str | Component | list[str | Component] | None,
         examples: list[Any] | list[list[Any]] | str | None = None,
-        cache_examples: bool | None = None,
+        cache_examples: bool | Literal["lazy"] | None = None,
```

Comment:
We need to update this typehint and doc for chat_interface!

Reply:
Right! Will add